### ISSUE TITLE
"This site appears to use a scroll-linked positioning effect. This may not work well with asynchronous panning"

### ISSUE THREAD

**Issue by charlwillia6** [fullcalendar/fullcalendar#4087]
```
15:32:48.195 This site appears to use a scroll-linked positioning effect. This may not work well with asynchronous panning; see https://developer.mozilla.org/docs/Mozilla/Performance/ScrollLinkedEffects for further details and to join the discussion on related tools and features!
job_scheduler
```

While I know that this isn't really a big "issue," I think that it needs to be brought to attention.  It appears that Firefox is recommending that all scroll-linked position effects be handled by CSS instead of being handled by JavaScript when using asynchronous handling, which is what I am actually doing.  It is causing performance issues in Firefox, so this may be of future concern.

**Comment by acerix** [comment]
Thanks for the report.

I don't see that warning in Firefox when testing the homepage demo, although I see this one in Chrome which sounds related, but seems to be an issue in jquery:

`jquery.min.js:3 [Violation] Added non-passive event listener to a scroll-blocking 'touchstart' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952`

Could you please specify the steps to get that warning, and which version of Firefox.  

Also, would you be able to post a [runnable, stripped-down demonstration of the performance issue](http://fullcalendar.io/wiki/Reduced-Test-Cases/)? Would really appreciate it because the time saved reproducing will be time spent fixing.

**Comment by arshaw** [comment]
i haven't seen or tried to recreate the warning, but i can trust that its happening and understand why. probably has to do with Scheduler's timeline event-title effect. I plan to use position:sticky in the future (via this: https://github.com/dollarshaveclub/stickybits) to solve.

when exactly does this error happen? with what view?

**Comment by Scrabble96** [comment]
I have this issue because I have a jquery based go-to-top button that appears when the site visitor scrolls down beyond a set number of pixels and disappears when they either click the button to go to the top of the page or scrolls back up again past the set position. I don't think that position:sticky would be a suitable alternative.

**Comment by arshaw** [comment]
this issue will be solved when these three issues are attacked (for v4):
https://github.com/fullcalendar/fullcalendar/issues/4149
https://github.com/fullcalendar/fullcalendar/issues/4311
https://github.com/fullcalendar/fullcalendar-scheduler/issues/480
&uarr;
Could you please visit the links, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates?

**Comment by arshaw** [comment]
released in [3.10.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v3.10.0)
